documentation/book/assembly-setting-up-tracing-mirror-maker-connect.adoc

```diff
@@ -0,0 +1,13 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-tracing.adoc
+[id='assembly-setting-up-tracing-mirror-maker-connect-{context}']
+= Setting up tracing for Mirror Maker and Kafka Connect
+
+Distributed tracing is supported for Mirror Maker and Kafka Connect (including Kafka Connect with Source2Image support). 
+
+For Mirror Maker, messages are traced from the source cluster to the target cluster; the trace data visualize messages entering and leaving the Mirror Maker component.
+
+Only messages produced and consumed by Kafka Connect itself are traced. To trace messages sent between Kafka Connect and external systems, you must configure tracing in the connectors for those systems.
```

Comment:
Would we want to link to more info on connector config?

Reply:
Added a link to Kafka Connect cluster configuration.